LogMonitor/src/LogMonitor/ProcessMonitor.cpp

```diff
@@ -121,6 +125,9 @@ DWORD CreateChildProcess(std::wstring& Cmdline)
                              &siStartInfo,  // STARTUPINFO pointer
                              &piProcInfo);  // receives PROCESS_INFORMATION
 
+    g_processId = piProcInfo.dwProcessId;
+    g_processName = Cmdline;

```

Comment:
Should we support processName as input or take the last <path>/processName.exe as the name to use here - feels like { "ProcessName": "path/to/process/executable" } may not make the most filterable value

Reply:
Yes, I got blocked on this since the exe image it not necessarily the process name. I wanted to find a way of getting the pid then name, but proved to need another extra function. I decided to leave this for now, until the users ask for it. Now that the entry command  / process is already known at the beginning.